src/lib/queries/getProcessDefinitions.handlers.ts

```diff
@@ -0,0 +1,91 @@
+import { rest, RestHandler } from 'msw'
```

Comment:
can you move this into a subfolder?
`src/lib/queries/getProcessDefinitions/getProcessDefinitions.handlers.ts`